### PR TITLE
sylius/refund-plugin recipe improvements

### PR DESCRIPTION
**Pull request by Zales0123**
| Q             | A
| ------------- | ---
| License       | MIT

It is meant to automatize 3rd point from [installation instruction](https://github.com/Sylius/RefundPlugin/blob/master/README.md#installation).

### CODE REVIEW

File: sylius/refund-plugin/0.4/manifest.json [hunk 1]
```diff
@@ -1,8 +1,18 @@
 {
     "bundles": {
+        "Sylius\\Bundle\\CoreBundle\\SyliusCoreBundle": ["all"],
+        "Sylius\\Bundle\\GridBundle\\SyliusGridBundle": ["all"],
+        "Sylius\\Bundle\\MailerBundle\\SyliusMailerBundle": ["all"],
+        "Sylius\\Bundle\\ResourceBundle\\SyliusResourceBundle": ["all"],
```
Comment (Nyholm) [inline sylius/refund-plugin/0.4/manifest.json]: These bundles is not part of the package. We should never modify other bundles in this recipe.

**Comment by Nyholm** [comment]
Any updates here?

**Comment by Nyholm** [comment]
Friendly ping

**Comment by Zales0123** [comment]
Sorry for the long response time, I've totally missed it :/ I had no time to work on this for some time recently, but for sure I want to make it work and will try as fast as possible :)

**Comment by Zales0123** [comment]
Ok, so it's been a long time, sorry for that 🖖 Right now it won't work, as the plugin requires Sylius to work and Sylius does not have a recipe defined... So I'll allow myself to close it (in the end, it's just an improvement), to not litter the PR's list and when we would have a recipe for Sylius and/or its bundles itself (I hope sooner than later ;))  I'll come back with it. Thanks a lot for Flex and Symfony! 🚀

**Pull request closed without merging**